# use correct version check for samtools
# as in the latest (unfortunatelly unreleased) version
diff -ruN dDocent-2.7.8/dDocent dDocent-2.7.8/dDocent
--- dDocent-2.7.8/dDocent	2020-11-03 16:27:30.032674965 +0100
+++ dDocent-2.7.8-new/dDocent	2020-11-03 18:00:48.750674872 +0100
@@ -28,8 +28,8 @@
 	fi
 done
 
-SAMV1=$(samtools 2>&1 >/dev/null | grep Ver | sed -e 's/Version://' | cut -f2 -d " " | sed -e 's/-.*//' | cut -c1)
-SAMV2=$(samtools 2>&1 >/dev/null | grep Ver | sed -e 's/Version://' | cut -f2 -d " " | sed -e 's/-.*//' | cut -c3)
+SAMV1=$(samtools 2>&1 >/dev/null | grep Ver | sed -e 's/Version://' | cut -f2 -d " " | sed -e 's/-.*//' | cut -f1 -d ".")
+SAMV2=$(samtools 2>&1 >/dev/null | grep Ver | sed -e 's/Version://' | cut -f2 -d " " | sed -e 's/-.*//' | cut -f2 -d ".")
 	if [ "$SAMV1"  -ge "1" ]; then
 		if [ "$SAMV2"  -lt "3" ]; then
         	echo "The version of Samtools installed in your" '$PATH' "is not optimized for dDocent."
Binary files dDocent-2.7.8/.dDocent.swp and dDocent-2.7.8-new/.dDocent.swp differ
